<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditSchemeDialog</class>
 <widget class="QDialog" name="EditSchemeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Фон</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="EditBrushWidget" name="widgetBackground" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Координаты</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,1">
       <item row="0" column="0" rowspan="4">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QListView" name="listView">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Центр</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>X</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="spinBox_3">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Y</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="spinBox_4">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Сетка</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_13">
            <item>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Начало по X</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="spinBox_5">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Начало по Y</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="spinBox_6">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="Line" name="line_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Шаг по X</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="spinBox_7"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Шаг по Y</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="spinBox_8"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="Line" name="line_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_11">
              <property name="text">
               <string>Видимая</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_12">
              <property name="text">
               <string>Привязка</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>453</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Установить матрицу</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>129</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Окно</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Прокрутка</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>Автоматический предел прокрутки</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_2">
            <property name="text">
             <string>Горизонтальная прокрутка</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_5">
            <property name="text">
             <string>Вертикальная прокрутка</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>354</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>354</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Размер</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QComboBox" name="comboBox_2">
            <item>
             <property name="text">
              <string>По умолчанию</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Максимизированное</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Минимизированное</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>По размеру пространства</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Установить</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>IШирина</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Высота</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_2">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_7">
            <property name="text">
             <string>Нельзя менять</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_8">
            <property name="text">
             <string>Автоопределение начала</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="2">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>354</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>128</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Стиль</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>По умолчанию</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Диалог</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Всплывающее окно</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Мультидокументное окно</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_6">
            <property name="text">
             <string>Модальное</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Разное</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QCheckBox" name="checkBox_9">
         <property name="text">
          <string>Запретить работу подокон</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_10">
         <property name="text">
          <string>Можно выбирать невыбираемые объекты</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>408</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Слои</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0,1">
       <item>
        <widget class="QGroupBox" name="groupBoxGraphicsLayer">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QCheckBox" name="checkBoxGraphicsLayers">
            <property name="text">
             <string>Графика</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="tristate">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidgetGraphicsLayers"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxObjectLayer">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QCheckBox" name="checkBoxImageLayers">
            <property name="text">
             <string>Имиджи</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="tristate">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidgetImageLayers"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxLinkLayer">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QCheckBox" name="checkBoxLinkLayers">
            <property name="text">
             <string>Связи</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="tristate">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidgetLinkLayers"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>4</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>Инструменты</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QTreeView" name="treeView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Объекты</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QTreeView" name="treeView_2"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EditBrushWidget</class>
   <extends>QWidget</extends>
   <header>EditBrushWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditSchemeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditSchemeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
